jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java

```diff
@@ -70,9 +70,15 @@ public static Map<String, Object> execute(Map<String, String> params) throws Nam
             break;
         }
         ArrayList errors= (ArrayList) errorInfo.get("errors");
+
         if(!errors.isEmpty()){
             //Below logger message is used by datadog to create notification in slack
             log.error("Error occurred in atleast one collector for jobId : {}",jobHint);
+            errors.stream().forEach(error -> {
+                Map<String,Object> err = (Map<String,Object>)error;
+                log.error("error in Main - {}",err.get("error"));
+                log.error("exception in main - {}",err.get("exception"));
```

Comment:
logging consistency

exception in main - {} -> Exception in Main : {}

Reply:
These logger statements are temporarily included to debug qualys data alert issue. They will be reverted once the issue is resolved. In qualys, we are getting data alert but when we check logs, we are not finding any error/exception which caused this data alert. To know what error caused this, we have included the above logger statements.

Reply:
done.